.circleci/config.yml

```diff
@@ -26,12 +31,27 @@ jobs:
             docker tag quay.io/giantswarm/cluster-api-controller-amd64:$CIRCLE_SHA1 registry-intl.cn-shanghai.aliyuncs.com/giantswarm/cluster-api-controller:$CIRCLE_SHA1
             docker tag quay.io/giantswarm/kubeadm-control-plane-controller-amd64:$CIRCLE_SHA1 registry-intl.cn-shanghai.aliyuncs.com/giantswarm/kubeadm-control-plane-controller:$CIRCLE_SHA1
             docker tag quay.io/giantswarm/kubeadm-bootstrap-controller-amd64:$CIRCLE_SHA1 registry-intl.cn-shanghai.aliyuncs.com/giantswarm/kubeadm-bootstrap-controller:$CIRCLE_SHA1
+      - run:
+          name: Re-tag images for git tags
+          command: |
+            if [[ -n $CIRCLE_TAG ]]; then docker tag quay.io/giantswarm/cluster-api-controller-amd64:$CIRCLE_SHA1 quay.io/giantswarm/cluster-api-controller:$CIRCLE_TAG; fi
```

Comment:
This line here, basically
```
if [[ -n $CIRCLE_TAG ]]; then <tag docker image>; fi
```

is added so we get Docker image tags for git tags/releases, as we currently only have Docker image tags for git commit SHAs.

Do we have a CircleCI guru who knows how to do this in a more elegant and less bashy way? I tried it with CircleCI filters and couldn't achieve it.